.github/workflows/SV.yml

```diff
@@ -1,13 +1,9 @@
 name: Schema Validation Check
 on:
   push:
-    branches:
-      - main
     paths:
       - "**/metadata.json"
```

Comment:
I think you can just remove this to just make the check pass if there's no metadata file

Reply:
That change makes sense, I’ll put that in place this evening. 

About it not working, it clearly does not. Look at any dependabot PR, always fails because the main branch is a protected branch

Reply:
Well for sure that's because it's doesn't contains any changes to metadata file since it only run if the PR contains the changes to metadata file

![image](https://user-images.githubusercontent.com/54318514/145540208-6d5c6bd1-05f9-4e3b-b2fb-9d3cabec4e36.png)

Reply:
I have no clue what you're on about. The idea here is to make it so the workflow **doesn't** run on PRs that **don't** edit the metadata files, it won't produce any errors, it simply **doesn't run**

Reply:
I mean make it so just that you won't have to force merge because the check is required

Reply:
That’s the whole point of this

Reply:
Removing the `branches` field won't do anything, it will just make it so it can run on any branches

Reply:
branches cheeks for /ref/head so the branch it comes from so it is not useful for our use